lib/rouge/lexers/rust.rb

```diff
@@ -100,6 +100,7 @@ def macro_closed?
 
         rule %r/([.]\s*)?#{id}(?=\s*[(])/m, Name::Function
         rule %r/[.]\s*#{id}/, Name::Property
+        rule %r/[.]\s*\d+/, Name::Property
```

Comment:
Given that the tuple index is functioning in a manner more akin to a property here, it felt more correct to use the same token as we would for accessing a property. What do you think?

Reply:
I was on the fence on this one. Semantically, I see why `Name::Property` makes sense. I went with `Num::Integer` mostly for the number syntax highlighting, since that's what my editor's syntax highlighting does.

Reply:
Hmmm. How about `Name::Attribute`? If we did that, it would allow people to set a different syntax colour for that if they wanted visual differentiation.

Reply:
That sounds OK to me. Should I make the change, or will you do it?

Reply:
I've got it open. I'll do it and push.